servers.json

```diff
@@ -1731,5 +1731,15 @@
         "primaryAddress": "battle.rip",
         "minecraftVersions": ["1.7", "1.8", "1.12"],
         "primaryMinecraftVersion": "1.7"
-    }
+    },
+    {
+        "id": "zeronmc",
+        "name": "ZeronMC",
+        "addresses": [
+            "zeronmc.ml"
+        ],
+        "primaryAddress": "play.zeronmc.ml",
```

Comment:
```suggestion
        "primaryAddress": "zeronmc.ml",
```

Reply:
oh sorry

Reply:
Sorry, we've made a mistake. Yes, you are able to keep your subdomain in the primaryAddress field.